README.md

```diff
@@ -6,4 +6,5 @@
 - Jason Beutler
 - Victor Navarro
 - Ashray Thapa
-- Brandon Dombrowsky 
\ No newline at end of file
+- Brandon Dombrowsky
+- TGIF
```

Comment:
Nice for testing, but since we're doing an "official" review process: do we need this line? (I'll approve if you want it here)

Reply:
I know it seems tedious, but if you would, that would be great. Here are three things I am checking:
1. Will a change this small and seemingly innocuous merge automatically?
2. How will the merge control box change? Will we get the green manual merge button?
3.  Though I have added as many restrictions as I can to prevent me from becoming an evil genius on this project, I STILL have an override option: 
![image](https://user-images.githubusercontent.com/91507453/212481691-b6749dcf-6032-4cad-a43e-33f118d77539.png)
I want to see if 2 approvals makes that go away.

Reply:
roger dodger